controllers/apply/lib/field-types/address.js

```diff
@@ -10,16 +10,16 @@ class AddressField extends Field {
 
         this.type = 'address';
 
-        if (!props.explanation) {
-            this.explanation = this.localise({
-                en: `<p>Enter the postcode and search for the address, or enter it manually below.`,
-                cy: `Rhowch y cod post a chwiliwch am y cyfeiriad, neu ei deipio isod.`
-            });
-        }
+        this.isRequired = true;
+
+        this.explanation = this.localise({
+            en: `Enter the postcode and search for the address, or enter it manually below.`,
+            cy: `Rhowch y cod post a chwiliwch am y cyfeiriad, neu ei deipio isod.`
+        });
     }
 
     defaultSchema() {
-        const schema = Joi.object({
+        return Joi.object({
```

Comment:
We never have an optional address lookup (not even sure if that'd work) so have made it always required.